client/src/services/stardust/stardustApiClient.ts

```diff
@@ -216,18 +217,18 @@ export class StardustApiClient extends ApiClient {
      * Download transaction history.
      * @param network The network in context.
      * @param address The address to download history for.
-     * @param timespan The timespan to download.
+     * @param targetDate The date to use.
      * @returns The history.
      */
     public async transactionHistoryDownload(
         network: string,
         address: string,
-        timespan: string
+        targetDate: moment.Moment
```

Comment:
Maybe we can use targetData as string so we don't have to import the moment lib just because of this one date.